JASP-Desktop/widgets/jaspcontrolwrapper.h

```diff
@@ -53,11 +55,13 @@ class JASPControlWrapper
 
 
 	static JASPControlWrapper*	buildJASPControlWrapper(JASPControlBase* control);
+
+	virtual QString				friendlyName() const { return ""; }
 	
 protected:
 	
-	JASPControlBase*				_item;
-	QVector<JASPControlWrapper*>	_depends;
+	JASPControlBase	*	_item = nullptr;
+	Set					_depends; //So Joris changed this to a set instead of a vector because that is what it seemed to be, the order isn't important right?
```

Comment:
OK

Reply:
Ok good! I put the comment here to make sure you double-checked me because I figured there might be more going on than I though. (I changed the type of `_depends` several times :p)